src/main/resources/db/migration/common/V2024.06.05.0001__conversation_prison_number_index.sql

```diff
@@ -0,0 +1,6 @@
+DROP INDEX conversation_prison_number_idx;
```

Comment:
This was a unique index which meant that a prisoner couldn't have multiple conversations. This drops the unique constraint.